### DISCUSSION TITLE
Cannnot open .gif

### DISCUSSION THREAD

**Discussion by Glenta** [d2phap/ImageGlass#752] [General] [answered]
### System information:
 - Windows OS version: Windows 10 Pro 1909
 - ImageGlass version: 7.5.1.1

### To Reproduce
1. Install ImageGlass with msi installer
2. Open a .gif
3. You get "ImageGIass cannot open this picture because the file appears to be damaged, corrupted or not supported. ImageGIass: Object reference not set to an instance of an object."

### Actual behavior:

Getting "ImageGIass cannot open this picture because the file appears to be damaged, corrupted or not supported. ImageGIass: Object reference not set to an instance of an object."

### Expected behavior:

Open the .gif

### Screenshots:

![image](https://user-images.githubusercontent.com/38775826/80106344-be4b4e80-8547-11ea-8363-96aea98a0082.png)

**Comment by d2phap** [comment]
Can you share your gif file, thank you

**Comment by Glenta** [comment]
While all gifs don't work, here are some examples
![wly3l6y50eh41](https://user-images.githubusercontent.com/38775826/80112015-9a3f3b80-854e-11ea-80d2-01b1c0926be9.gif)
![k82cy41qshx31](https://user-images.githubusercontent.com/38775826/80112077-b04cfc00-854e-11ea-9674-e6bc9e4b53bb.gif)

**Comment by d2phap** [comment]
@Glenta 
Your gifs don't have the correct header. It's not only ImageGlass, Chrome also cannot open it

**Comment by fire-eggs** [comment, marked as answer]
To confirm: those are GIF files, they're video files. You can play them back with a video player such as VLC, MPV,  or Windows Media Player. Rename them from .GIF to .MP4 and you'll be good to go.

Those files look like they might be from Imgur. For some reason, a lot of people use the term "GIF" to refer to videos. In fact, I believe that if someone actually posts a GIF file to Imgur, it is converted to a video file.

The following is an actual GIF file:

![scooter_dribbble](https://user-images.githubusercontent.com/9809727/80159146-48bb9e80-8598-11ea-9864-57ddd09efaea.gif)
